docs/demos/credit-card-fraud.md

```diff
@@ -0,0 +1,117 @@
+- **Cluster Storage:** Create new persistent storage - I call it "Credit Fraud Storage" and set the size to 20GB.
+
+Press Create Workbench and wait for it to start - status should say "Running" and you should be able to press the Open link
+![Workbench Settings](img/Workbench_Settings.png)
+![Workbench](img/Workbench.png)
+
+Open the workbench and login if needed.
+
+### 3: Train the model
+When inside the workbench (Jupyter), pull down this repo from GitHub: [https://github.com/red-hat-data-services/credit-fraud-detection-demo](https://github.com/red-hat-data-services/credit-fraud-detection-demo).   
+It contains:
+
+- A notebook (model.ipynb) inside the `model` folder with a Deep Neural Network model we will train.
+- Data for that model.
+- An application (model_application.py) inside the `application` folder that will fetch the trained model from MLFlow and run a prediction on it whenever it gets any user input.
+![Jupyter](img/Jupyter.png)
+
+Take a look inside the folders and scripts if you are interested.  
+Some things to note is how we set up tracking and logging with MLFlow in `model.ipynb` to make sure that the experiment and the model is captured. You can also read about that in the [MLFlow guide](/tools_and_applications_mlflow#adding-mlflow-to-training-code).
+
+When you are done, open up the `model.ipynb` notebook and run all the cells. If everything is set up correctly it will train the model and push both the experiment and the model to MLFlow.  
+The experiment is a record with metrics of how the run went, while the model is the actual tensorflow model which we later will use for inference.  
+You may see some warnings in the last cell related to MLFlow, as long as you see a final progressbar for the model being pushed to MLFlow you are fine:
+![Trained model](img/Trained_model.png)
+
+### 4: View the model in MLFlow
+Let's take a look at how it looks inside MLFlow now that we have trained the model.  
+If you opened the MLFlow UI in a new tab in step 1.1, then just swap over to that tab, otherwise follow these steps:
+
+- Go to the OpenShift Console
+- Make sure you are in Developer view in the left menu
+- Go to Topology in the left menu
+- At the top left, change your project to MLFlow (or whatever you called it when installing the MLFlow operator in pre-requisites)
+- Press the "Open URL" icon in the top right of the MLFlow circle in the topology map
+
+![Open MLFlow UI](img/Open_MLFlow_UI.png)
+
+When inside the MLFlow interface you should see your new experiment in the left menu. Click that to see all the runs, should be a single run from the model we just trained.  
+You can now click on the row in the Created column to get more information about the experiment and how to run the model.
+
+![MLFlow view](img/MLFlow_view.png)
+
+### 5: Deploy the model
```

Comment:
You should give more general info on what is going to happen. We don't know why we pull things from git.
You must at least explain that the application folder contains the code to serve the model from MLFlow, and that OpenShift will build a container image automatically from this code and publish it.

Reply:
Added a section explaining what it is we are going to be doing before jumping into how we do it